README_BUILD.md

```diff
@@ -1,45 +1,56 @@
-Building DSC Resources
-======================
-## Quick-start
+# Building DSC Resources
 
-~~~
-# install the bundler gem
-gem install bundler
+The `dsc` module uses rake tasks to parse the DSC Resource MOF schema files to generate Puppet types. This process pulls the DSC Resources from the github [PowerShell/DSCResources](https:/github.com/powershell/DSCResources) repo using git. This is a point-in-time snapshot, so anytime any of those DSC Resources are updated the build process needs to be run again to pull in the updated resources and parse them. If one wants to use non Microsoft DSC Resources, then you should follow the [Build Custom DSC Resource Types](#building-puppet-types-from-custom-dsc-resources) instructions.
 
-# install the needed ruby gems/libs
-bundle install
+## Requirements
+
+* This module must be built with a **non-Windows** computer due to limitations in the gems that the builder uses to generate types and specs.
+* Git client needs to be version 2.2.0 or above
 
-# Run the build and tests (this will remove all puppet types and build them again)
-bundle exec rake
-~~~~
+## Setup
 
-## Rake tasks
-You can use following rake tasks for your convenience
+To build the dsc module, you need to clone the source from our github repo and bundle install all the gem dependencies.
 
-~~~
-rake dsc:build             # Import and build all
-rake dsc:clean             # Cleanup all
-rake dsc:dmtf:clean        # Cleanup DMTF CIM MOF Schema files
-rake dsc:dmtf:import       # Import DMTF CIM MOF Schema files
-rake dsc:resources:clean   # Cleanup DSC Powershell modules files
-rake dsc:resources:import  # Import DSC Powershell modules files
-rake dsc:types:build       # Build DSC types in (lib/puppet/type)
-rake dsc:types:clean       # Cleanup DSC types in (lib/puppet/type)
+~~~bash
+# clone the dsc repo
+git clone https://github.com/puppetlabs/puppetlabs-dsc
+# change directory to the code
+cd puppetlabs-dsc
+# install the needed ruby gems/libs
+bundle install --path .bundle/gems --without system_tests
 ~~~
 
-## Building
+*NOTE*: This guide assumes that you have a working ruby installation with bundler installed. The latest version of ruby is assumed, as of this writing this guide was tested with 1.9.3 and 2.3.1.
```

Comment:
Sentence doesn't make sense.  Perhaps simply state that this guide was tested with Ruby 1.9.3 and 2.3.1

Didn't we (puppet) EOL ruby 1.9 anyway?